.changeset/polite-walls-rush.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': major
+'@itwin/itwinui-css': major
+---
+
+Updated Tile to be composition of customizable subcomponents
```

Comment:
Can you separate changeset files for css and react changes? This way you could be more descriptive with what is changed, eg. in css the main change is folder tile changing to display: grid.

Reply:
I created 2 separate changesets

Reply:
changeset description should be useful